### DISCUSSION TITLE
Azure Move from managed to self-hosted developer portal blob storage field missing in srs/config* files and delegation not working

### DISCUSSION THREAD

**Discussion by ShubjeetPal** [Azure/api-management-developer-portal#2635] [General]
**Every bug report should have precise description and reproduction steps; console traces or source code references are appreciated.**

**For assistance requests, contact Azure support or submit a post [on Stack Overflow](http://aka.ms/apimso). We don't provide support through GitHub Issues. Feature requests can be raised [on the Azure Feedback Forum](https://aka.ms/apimwish).**

## Bug description

The azure document says as below for "Move from managed to self-hosted developer portal" but their is no field in the srs/config* files

`The storage account in the config.design.json file needs to be the same as the storage account of the managed version of the portal`

The document also does not mention on what needs to be done for delegation as currently it fails with 404

## Reproduction steps

1. Go to 'https://docs.microsoft.com/en-us/azure/api-management/developer-portal-self-host'

## Expected behavior

1. Update of azure documents for blob storage fields
2. Update of srs/config* files for the optional parameters of blob storage in "config.design.json" and "config.publish.json" with comments as optional and describing what values to enter
3. Update of azure documents what is required for existing delegation to work and test it for example API monetization

## Is your portal managed or self-hosted?

Self-hosted

## Release tag or commit SHA (if using self-hosted version)

NA

## API Management service name

NA

## Environment

NA

## Additional context

NA

**Comment by ghost** [comment]
@ShubjeetPal, thank you for opening this issue. We will triage it within the next few business days.

**Comment by azaslonov** [comment]
@ShubjeetPal, delegation requires portal's backend to generate signed delegation URL. You can do so by specifying "backendUrl"  setting (pointing to your APIM service) in config.json file of your published website (which you upload to blob storage). However, if you do it right now, you'll get CORS errors which you'll be able to overcome in next few weeks with new CORS settings feature that is being rolled out.

**Comment by ShubjeetPal** [comment]
Thanks @azaslonov. Can you please provide the link for the new CORS setting feature.
Is this new CORS feature for both self-hosted and managed portal. As Managed portal has also delegation issues which you might be already aware of. https://github.com/Azure/api-management-developer-portal/issues/1479
Also can you update on the "Move from managed to self-hosted developer portal" issue which I have mentioned in my bug description section.
